SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -1258,6 +1258,23 @@ class CSolver {
                                               CNumerics *visc_numerics,
                                               CConfig *config,
                                               unsigned short val_marker) { }
+  
+   /*!
+    * \brief A virtual member.
+    * \param[in] geometry - Geometrical definition of the problem.
+    * \param[in] solver_container - Container vector with all the solutions.
+    * \param[in] conv_numerics - Description of the numerical method.
+    * \param[in] visc_numerics - Description of the numerical method.
+    * \param[in] config - Definition of the particular problem.
+    * \param[in] val_marker - Surface marker where the boundary condition is applied.
```

Comment:
I know we have this for most boundaries, but let me give you something to think about. This "a virtual member" is useless from the point of view of documenting the interface.
The place where functions are declared virtual defines the interface of the class, the basic purpose of this function should be stated here, then, when the function is overridden by derived classes they should only document what is specific of that class, towards implementing what the interface exposes... but anyway SU2 being SU2.